.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: npm CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  set_destination:
+    runs-on: self-hosted
+    environment: NEW_ER_CLOUD
+    outputs: 
+        destination: ${{ steps.set_destination.outputs.destination }}
+    steps:
+      - name: Set Destination
+        id: set_destination
+        run: |
+          echo "destination=\\\${{ vars.WEB_HOST }}\R9Web\ExperienceBuilder\client\" >> $GITHUB_OUTPUT
+
+  
+  build:
+
+    runs-on: self-hosted
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }} and output directory ${{ GITHUB.OUTPUT }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build:dev --if-present
```

Comment:
--if-present? :prod and :dev? I thought there was just one build that fed into the EB dev on the ER cloud. 
If there is a PR/staging setup it would probably just be for unit test (not applicable at the moment) unless we had a second EB dev setup.

Reply:
original Jenkins file is running both build:dev and buld:prod, so I am copying that. if-present seemed like a good way to prevent build failure if one of those scripts did not exist since it's all one environment.

`
        dir("D:/R9Web/ArcGISExperienceBuilder/client") {
            bat "npm i"
            bat "npm run build:dev"
            bat "npm run build:prod"
        }
`

Reply:
Ok then... Do we know what the implications of not running one of them would be? Seems to me if they are both required (there is no second environment) failure in the action would be preferred so we know something went wrong.  Let's move forward assuming both are required and drop the --if-present.